src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -719,4 +719,7 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
     <value>Writing packages lock file at disk. Path: {0}</value>
     <comment>{0} - Path to packages lock file</comment>
   </data>
+  <data name="Error_NoMatchingCertificate" xml:space="preserve">
+    <value>The package signature did not match any of the allowed certificate fingerprints.</value>
```

Comment:
nit: 
Will the user know which package it's referring to?
I'm assuming that this message is accompanied by some other ones, but it's a bit harder to follow the whole code.

Reply:
Yes. In the case of restore, this message is accompanied by other ones that specify which package. In the case of verify command, the whole process runs once per package, so this message will be in the context of a single package, where there's no need to specify the package.